docs/how-to/configure-logging.md

```diff
@@ -0,0 +1,53 @@
+
+# Configure Logging
+
+By default BlueAPI will log to stdout at the [INFO level](https://docs.python.org/3/library/logging.html#logging-levels), but can be reconfigured to log at any level, and to output to Graylog.
+
+When logging to [Graylog](https://graylog.org) is enabled, BlueAPI will also continue to log to stdout.
+
+# BlueAPI Cofiguration
+
+:::{seealso}
+[Configure the Application](./configure-app.md)
+:::
+
+An example logging config is shown below:
+```
+  logging:
+    level: "INFO"
+    graylog:
+      enabled: True
+      host: "graylog-log-target.diamond.ac.uk"
+      port: 12232
+```
+
+Here, BlueAPI will accept all messages are INFO level and above and will forward these to both the referenced Graylog address, and to stdout.
+
+# Instrumenting Libraries
+To instrument to a custom module in BlueAPI, instantiate a logger from the [standard library logging package, then use any of its log methods](https://docs.python.org/3/library/logging.html#logger-objects).
+
+BlueAPI is written to intercept logs from any python code it executes.
```

Comment:
```suggestion
BlueAPI is configured to handle logging from any python code it executes.
```

This is a bit picky but I'm not aware of blueapi doing any logging handling/intercepting itself.